types/jsdoc-to-markdown/v4/index.d.ts

```diff
@@ -0,0 +1,137 @@
+    /**
+     * The template the supplied documentation will be rendered into.
+     * Use the default or supply your own template for full control over the output.
+     */
+    template?: string;
+    /**
+     * The initial heading depth.
+     * For example, with a value of 2 the top-level markdown headings look like "## The heading".
+     */
+    headingDepth?: number;
+    /**
+     * Specifies the default language used in '@example' blocks (for syntax-highlighting purposes).
+     * In gfm mode, each '@example' is wrapped in a fenced-code block. Example usage: --example-lang js.
+     * Use the special value none for no specific language.
+     * While using this option, you can override the supplied language
+     * for any '@example' by specifying the @lang subtag,
+     * e.g @example @lang hbs. Specifying @example @lang off will disable code blocks for that example.
+     */
+    exampleLang?: string;
+    /**
+     * Use an installed package containing helper and/or partial overrides.
+     */
+    plugin?: string|string[];
+    /**
+     * handlebars helper files to override or extend the default set.
+     */
+    helper?: string|string[];
+    /**
+     * handlebars partial files to override or extend the default set.
+     */
+    partial?: string|string[];
+    /**
+     * Format identifier names in the code style,
+     * (i.e. format using backticks or <code></code>).
+     */
+    nameFormat?: string;
+    /**
+     * By default, dmd generates github-flavoured markdown.
+     * Not all markdown parsers render gfm correctly.
+     * If your generated docs look incorrect on sites other than Github
+     * (e.g. npmjs.org) try enabling this option to disable Github-specific syntax.
+     */
+    noGfm?: boolean;
+    /**
+     * Put <hr> breaks between identifiers. Improves readability on bulky docs.
+     */
+    seperators?: boolean;
+    moduleIndexFormat?: StyleListFormat;
+    globalIndexFormat?: StyleListFormat;
+    /**
+     * Two options to render parameter lists: 'list' or 'table' (default).
+     * Table format works well in most cases but switch to list if things begin to look crowded / squashed.
+     */
+    paramListFormat?: RenderListFormat;
+    propertyListFormat?: RenderListFormat;
+    memberIndexFormat?: MemberIndexFormat;
+}
+
+export interface JsdocOptions {
+    /**
+     * By default results are cached to speed up repeat invocations.
+     * Set to true to disable this.
+     */
+    noCache?: boolean;
+    /**
+     * One or more filenames to process.
+     * Accepts globs (e.g. *.js). Either files, source or data must be supplied.
+     */
+    files: string|string[];
+    /**
+     * A string containing source code to process.
+     * Either files, source or data must be supplied.
+     */
+    source?: string;
+    /**
+     * The path to the jsdoc configuration file.
+     *  Default: path/to/jsdoc/conf.json.
+     */
+    configure?: string;
+}
+
+export class JsdocToMarkdown {
```

Comment:
I feel like I'm missing something obvious, but this class wasn't exported in v4 either, was it?

Reply:
@amcasey amended, yes, left the v4 without a revision as I was not sure what's the correct approach. Thx for reviewing!

Reply:
@peterblazejewicz We usually allow breaking changes without a version bump if the existing state is incorrect (e.g. won't actually work at runtime), so I think it makes sense to clean up the old one too.